draft-ietf-webtrans-http2.md

```diff
@@ -1171,6 +1171,50 @@ parameter (see {{h2-settings}}).  Clients can select the associated upgrade
 token, if applicable, to use when establishing a new session, ensuring that
 servers will always know the syntax in use for every incoming request.
 
+# Use Outside of HTTP/2
+
+## Other HTTP Versions
+
+This document specifies a protocol that provides WebTransport {{OVERVIEW}}
+functionality when transported using HTTP/2 {{HTTP2}}.  Because the
+mechanisms defined in this document rely only on generic HTTP semantics, they
+enable many WebTransport capabilities when using any transport that provides a
+bidirectional stream of bytes, including other HTTP versions.  However, this protocol MUST NOT be used within HTTP/3, as the protocol defined in {{WEBTRANSPORT-H3}} provides a higher fidelity mapping to the underlying transport, for example unreliable datagram capabilities.
+
+Clients that execute untrusted application code, such as Web browsers, MUST
+NOT use this protocol over HTTP/1, as there is a potential that the receiving
+Web server does not understand the WebTransport protocol.  Other clients MAY
+use this protocol over HTTP/1 by specifying the token `webtransport` as the value for the
+`Upgrade` header field.
+
+## Non-HTTP Protocols
```

Comment:
This seems like an incomplete specification of an HTTP-like thing.  Why?

Reply:
> This seems like an incomplete specification of an HTTP-like thing. Why?

There's no reason why the WebTransport API, specifically the WHATWG Streams implementations, can't be used for IPC, Unix domain sockets, Native Messaging.

Reply:
Raw TCP sockets, raw UDP sockets et al. The full-duplex streaming using WHATWG Streams parts is useful beyond TLS, HTTP/3 and QUIC.